dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
No, the mapfile also has library-prerelease, I used `sort`, so it's somewhere in the middle.
http://dlang.org/.dpl_rewrite_map.txt